### PR TITLE
Update dev and install instructions. Add webpack functionaliaties

### PR DESCRIPTION
**Pull request by jearmoo**

### CODE REVIEW

File: Documentation/Development/README.md [hunk 1]
```diff
@@ -1,31 +1,47 @@
...
+```
+
+### Edit src/frontend/static/index.html
+Change the line
+```html
+<script type="text/javascript" src="/bundle.js"></script>
+```
+to
+```html
+<script type="text/javascript" src="/build/bundle.js"></script>
```
Comment (blengerich) [inline Documentation/Development/README.md]: Why is this necessary?

**Comment by jearmoo** [comment]
That's where the development version of webpack outputs its bundle.js right now

**Comment by blengerich** [comment]
Ok, at some point we should streamline this be the same expected location. But this is not a blocking issue for merging.

Nice work, looks good.
